landing.css

```diff
@@ -0,0 +1,55 @@
+@import url('./css/main.css');
+
+body{
+    background-color: var(--Elevation-0);
+    background-color: var(--Dark-0);
+    color: var(--Light-0);
+    line-height: 1.4rem;
+}
+
+.navbar{
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+    gap: 1rem;
+    padding: 1.25rem 3rem;
+    color: var(--Light-3);
+    box-shadow: var(--Shadow-03);
+    background-color: var(--Elevation-3);
+    border-radius: 0rem 0rem .8rem .8rem;
+}
+
+.navbar-brand{
+    margin-right: auto;
+    margin-bottom: 0;
+    color: var(--Green-3);
+    font-size: 2rem;
+}
+.navbar nav{
+    margin-left: auto;
+}
+.small-text{
+    color: var(--Green-0);    
+}
+
+.hero-landing{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 10rem;
+    padding: 1rem 3rem;
+    height: 100%;
```

Comment:
it would be great if you give this like hieght: 100vh.

Reply:
will do that

Reply:
when I am giving height: 100vh. 
 it gets overflow so I made it as 80vh is that fine?